styles/toast.css

```diff
@@ -0,0 +1,43 @@
+.toast-container{
+    background-color: var(--text-color-black);
+    color: var(--body-background-color);
+    display: flex;
+    flex-direction: row;
```

Comment:
flex-direction is by default row

Reply:
in this case i have used flex center property where i have given the flex direction as column so did here